src/clndr.js

```diff
@@ -533,9 +534,13 @@
   Clndr.prototype.bindEvents = function() {
     var $container = $(this.element);
     var self = this;
+    var eventType = 'click';
+    if (self.options.useTouchEvents === true) {
+      eventType = 'touchstart';
```

Comment:
I did try that, with poor results on mobile - it would process both click and touch events, resulting in duplicate callbacks.